css-positioning/fixed.css

```diff
@@ -0,0 +1,12 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+
+  /* your code here */
+  position: fixed;
+  left: 25%;
+}
```

Comment:
You have this as `left: 25%` and `text-align: center`. What this does is push it 25% of width from the left side and then still centers the text. Instead you should make this element space from the _left_ to the _right_ side and then your `text-align: center` should center the header. 

** do not use `width` for this.